roles/python/vars/main.yml

```diff
@@ -1,10 +1,11 @@
 python_versions:
-  - 2.7
-  - 3.4
```

Comment:
Let's leave 3.4 there so we can use this release to test django 2.0 as well. The rest of the docs + config still expect 3.4 to exist also.

Reply:
Hmm, actually not sure if 3.4 will work with the 2.1 tox file. Maybe we should just remove 3.4 references from the config and docs here instead?

Reply:
Exactly, Django tests trunk post-2.0 don't run with Python 3.4 anymore. I saw the failures when smoke testing the re-generated image with `box-tests.sh` and that's what led me to also remove 3.4.

Reply:
Also, please ignore the Python version vars changes here. Work on that front is in #32. Hopefully we will be able to merge it first and then I will rebase this.